db/migrate/20190311204446_add_fields_to_medication_reports.rb

```diff
@@ -0,0 +1,6 @@
+class AddFieldsToMedicationReports < ActiveRecord::Migration[5.1]
+  def change
+    add_column :medication_reports, :took_medication, :boolean
```

Comment:
```suggestion
    add_column :medication_reports, :took_medication, :boolean, null: false
```

If you accept this suggestion, you'll have to:
```bash
docker-compose exec web rake db:rollback
docker-compose exec web rake db:migrate
```

Reply:
After you run those, you should see the `null: false` designation in `db/schema.rb`.